packages/ckeditor5-emoji/src/emojimention.ts

```diff
@@ -60,11 +61,11 @@ export default class EmojiMentionIntegration extends Plugin {
 	constructor( editor: Editor ) {
 		super( editor );
 
-		this.emojiDropdownLimit = editor.config.get( 'emoji.dropdownLimit' ) || DEFAULT_DROPDOWN_LIMIT;
-		this.mentionMarker = editor.config.get( 'emoji.marker' ) || DEFAULT_MENTION_MARKER;
+		this._emojiDropdownLimit = editor.config.get( 'emoji.dropdownLimit' ) || DEFAULT_DROPDOWN_LIMIT;
+		this._mentionMarker = editor.config.get( 'emoji.marker' ) || DEFAULT_MENTION_MARKER;
```

Comment:
By default, `editor.config.get( 'emoji' )` returns `undefined`. Even if the Emoji plugin is loaded. Hence, I propose using `editor.config.define( 'emoji', { /* ... */ } )` to set up initial values.